.tekton/determine-image-tag-task-oci-ta.yaml

```diff
@@ -41,4 +44,18 @@ spec:
       dnf -y install git make
 
       .konflux/scripts/fail-build-if-git-is-dirty.sh
-      echo -n "$(make --quiet --no-print-directory tag)$(params.TAG_SUFFIX)" | tee "$(results.IMAGE_TAG.path)"
+      image_tag=""
```

Comment:
This line is probably not necessary...

Reply:
I consider it good practice to properly initialize the used variables.

Reply:
I think in shell "initializing" does more harm than good, because unitialized one will show up as such with `set -u` while an empty one will often cause confusing behaviour. But meh.